include/api/CInferenceModelDefinition.h

```diff
@@ -247,6 +306,21 @@ class API_EXPORT CEncoding : public CSerializableToJson {
 //! \brief Mapping from categorical columns to numerical values related to categorical value distribution.
 class API_EXPORT CFrequencyEncoding final : public CEncoding {
 public:
+    class CSizeInfo final : public CEncoding::CSizeInfo {
+    public:
+        explicit CSizeInfo(const CFrequencyEncoding& encoding);
+        void addToDocument(rapidjson::Value& parentObject, TRapidJsonWriter& writer) const override;
+        const std::string& typeString() const override;
+
+    private:
+        using TSizeVec = std::vector<std::size_t>;
```

Comment:
nit: how about moving this typedef into protected `CEncoding::CSizeInfo` to save duplicating.

Reply:
Good catch!